### PR TITLE
clarify the execution contexts of hook handlers with different action

### PR DESCRIPTION
**Pull request by Cweiping**
Clarify how kubelet manages hooks.
Close Issue:https://github.com/kubernetes/website/issues/23670

### CODE REVIEW

**Comment by Cweiping** [comment]
Take your time to review this PR.Thx!
/assign @onlydole

**Comment by onlydole** [comment]
Thank you, @Cweiping! This looks good to me 👍🏼 

/lgtm

File: content/en/docs/concepts/containers/container-lifecycle-hooks.md [hunk 1]
```diff
@@ -56,7 +56,8 @@ Resources consumed by the command are counted against the Container.
 ### Hook handler execution
 
 When a Container lifecycle management hook is called,
-the Kubernetes management system executes the handler in the Container registered for that hook. 
+the Kubernetes management system will execute handler according to the hook action,
```
Comment (zacharysarah) [inline content/en/docs/concepts/containers/container-lifecycle-hooks.md:59]: Use [present tense](https://kubernetes.io/docs/contribute/style/style-guide/#use-present-tense):

```suggestion
the Kubernetes management system executes the handler according to the hook action,
```

File: content/en/docs/concepts/containers/container-lifecycle-hooks.md [hunk 2]
```diff
@@ -56,7 +56,8 @@ Resources consumed by the command are counted against the Container.
...
+`exec` and `tcpSocket` will be executed in the container, and `httpGet` will be executed by the kubelet process.
```
Comment (zacharysarah) [inline content/en/docs/concepts/containers/container-lifecycle-hooks.md:60]: Use [present tense](https://kubernetes.io/docs/contribute/style/style-guide/#use-present-tense):

```suggestion
`exec` and `tcpSocket` are executed in the container, and `httpGet` is executed by the kubelet process.
```

**Review by zacharysarah — CHANGES_REQUESTED** [review]
@Cweiping Thanks for this update. Minor changes required, otherwise LGTM

**Comment by yogendra** [comment]
Lgtm

**Comment by sftim** [comment]
@Cweiping with a few changes, this will be good to merge. Would you have time to revise this PR?

**Comment by Cweiping** [comment]
> @Cweiping with a few changes, this will be good to merge. Would you have time to revise this PR?

Of course, I'd like to

File: content/en/docs/concepts/containers/container-lifecycle-hooks.md (hunk 2 above)
Comment (Cweiping) [reply content/en/docs/concepts/containers/container-lifecycle-hooks.md:60]: Modify according to [present tense](https://kubernetes.io/docs/contribute/style/style-guide/#use-present-tense). 
Thank you for your advice,please review again.

**Review by irvifa — COMMENTED** [review]
Thanks!
/lgtm
/approve

**Pull request merged**